### ISSUE TITLE
macOS and Windows packages

### ISSUE THREAD

**Issue by davelab6** [MFEK/glif#219]
When do you plan to start including these in https://github.com/MFEK/glif/releases ? :)

**Comment by ctrlcctrlv** [comment]
Certainly, there cannot be a non-&beta; build of MFEKglif without some type of packaging for all 3 major platforms.

Whether I will do this on each module is unclear, but probably not; my ultimate idea is:

* Windows installer (`.msi`) that will install all MFEK modules to `C:\Program Files\Modular Font Editor K\` (MFEKmetadata.exe, MFEKstroke.exe, MFEKpathops.exe, MFEKglif.exe, MFEKufo.exe…)
* MacOS `.dmg` file that will have all MFEK modules in notarized binary form for extraction to `/Applications`
* Linux `.tar.xz` with all the binaries statically compiled for extraction to e.g. `$HOME/.bin`.

It is a project-wide problem therefore solution must also be project-wide, and not focused on one module, even if it's the most popular one.

Solving this issue will become more important the more user friendly the project as a whole is, at current iteration it's not so pressing as non-technical user unlikely to get much use out of the binaries anyway even if they have them; many only provide CLI API's, even MFEKglif when in graphical mode is very undocumented, etc.

**Comment by justvanrossum** [comment]
Success! I can view and edit a .glif!

This is god enough for me, but you may want to keep this open, as I still can't build from the sources.

**Comment by ctrlcctrlv** [comment]
@justvanrossum Did you reply to the wrong issue and you meant #220?

If so I'd rather close #220 as it's about the narrow bug you experienced with font titles, and leave this issue open to track progress of binary packaging, which your issue became less and less about over time :-)

**Comment by justvanrossum** [comment]
Yes, duh, I'm sorry! Will post to the correct one. You can delete here if you want.

**Comment by prajwalprabhu** [comment]
> Certainly, there cannot be a non-β build of MFEKglif without some type of packaging for all 3 major platforms.
> 
> Whether I will do this on each module is unclear, but probably not; my ultimate idea is:
> 
>     * Windows installer (`.msi`) that will install all MFEK modules to `C:\Program Files\Modular Font Editor K\` (MFEKmetadata.exe, MFEKstroke.exe, MFEKpathops.exe, MFEKglif.exe, MFEKufo.exe…)
> 
>     * MacOS `.dmg` file that will have all MFEK modules in notarized binary form for extraction to `/Applications`
> 
>     * Linux `.tar.xz` with all the binaries statically compiled for extraction to e.g. `$HOME/.bin`.
> 
> 
> It is a project-wide problem therefore solution must also be project-wide, and not focused on one module, even if it's the most popular one.
> 
> Solving this issue will become more important the more user friendly the project as a whole is, at current iteration it's not so pressing as non-technical user unlikely to get much use out of the binaries anyway even if they have them; many only provide CLI API's, even MFEKglif when in graphical mode is very undocumented, etc.

What if we write a gui app ,which will be a online installer.
And all the libraries we need will be compiled and kept in github release section and our app will fetch it from there and keep in resective directory depending on the os.

**Comment by ctrlcctrlv** [comment]
The Modular Font Editor K Foundation, Inc. has awarded @Subject38 a grant to fix this issue on MacOS. This is «MFEK Grant №3, Series of 2022».

(Note: The Foundation does not disclose individual award amounts for the privacy of our awardees; only the totals at the end of the year in the appropriate paperwork all charities have to file.)

**Comment by ctrlcctrlv** [comment]
Marked `on hold` pending outcome of issue I described in https://github.com/fontforge/fontforge/issues/5112#issuecomment-1368459352.

> I may as well give an update on where I am with solving this issue. I mentioned on April 21 that MFEK Fndn Inc. may be able to share keys (https://github.com/fontforge/fontforge/issues/4082#issuecomment-1105739888). That's not going to happen. Apple makes the process way too complicated, with many hoops to jump through.
> 
> I gave up on jumping through those hoops when I heard about a new European Union law, Regulation (EU) 2022/1925, The Digital Markets Act, which is already in force, but without any organization set up by the EU to approve alternative gatekeepers to installation on Apple devices, it doesn't enter into force in practice until May 1, 2023. And, Apple will likely do all they can to stymie the process, because they are Apple. Mother Theresa, Inc. could want to become an EU DMA gatekeeper and they'd oppose it, I'm sure.
> 
> I'll be watching with great interest the plans of the [Free Software Foundation Europe e.V](https://fsfe.org/activities/dma/dma.en.html#dma-your-help).
> 
> They seem to be planning to become a gatekeeper under the DMA.
> 
> Once this happens, I am sure they will have no issues registering a certificate for FontForge.

Or MFEK for that matter.

**Comment by davelab6** [comment]
May 1, 2023! That is soon! :)
